packages/types/src/NetworkInterfaces.ts

```diff
@@ -5,8 +5,23 @@ import { SecureContextConfig } from './StorageLinkInterfaces';
 export enum EnvironmentType {
```

Comment:
Can we start calling it `Network` rather than `EnvironmentType`? Because that's how we refer to it.

Of course, no need for a big refactoring right now on everything using `EnvironmentType`. Anyway, we'll have to keep it for backward compatibility for some time.

```ts
export enum Network {
  LOCAL = "local",
  DEVNET = "devnet", // or whatever we call it
  BANKSIA = "banksia",
  MYRTLE = "myrtle",
}

export enum EnvironmentType {
  LOCAL = Network.LOCAL,
  DEVNET = Network.DEVNET,
  BANKSIA = Network.BANKSIA,
  MYRTLE = Network.MYRTLE,

  /**
   * @deprecated use MYRTLE instead
   */
  MAINNET = Network.MYRTLE,
  /**
   * @deprecated use BANKSIA instead
   */
  TESTNET = Network.BANKSIA,
}
```

Reply:
Done